fawltydeps/main.py

```diff
@@ -76,6 +75,18 @@ def is_enabled(self, *args: Action) -> bool:
         """Return True if any of the given actions are in self.settings."""
         return len(self.settings.actions.intersection(args)) > 0
 
+    @staticmethod
+    def success_message(check_undeclared: bool, check_unused: bool) -> str:
+        """Returns the message to print when the analysis finds no errors."""
+        checking = []
+        if check_undeclared:
+            checking.append("undeclared")
+        if check_unused:
+            checking.append("unused")
+        if checking:
+            return f"No {' or '.join(checking)} dependencies detected."
+        return ""
+
```

Comment:
Nit: Does it make sense to return `Optional[str]` and `return None` in the fallthrough case, to more clearly identify the no-message case?

Reply:
Nit is is, but I agree :)

Reply:
Done in [3ea141c](https://github.com/tweag/FawltyDeps/pull/230/commits/3ea141cb794a71c33d1cc205e1c100255c8d8e91).